src/config.py

```diff
@@ -0,0 +1,40 @@
+
+import os
+
+
+class Config:
+    '''
+    Configuration for the application.
+
+    Any values that need to be exposed through Flask MUST be in `UPPERCASE`.
+    '''
+
+    # Flask Configuration
+
+    HOST: str = '0.0.0.0'
```

Comment:
Not sure what the use case for changing the host name would be currently.

Reply:
These settings are forwarded to Flask, and there is a use case i.e. testing on `localhost` only.